StackExchange.Redis/StackExchange/Redis/Interfaces/IDatabase.cs

```diff
@@ -1389,6 +1389,250 @@ public interface IDatabase : IRedis, IDatabaseAsync
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>The messages successfully claimed by the given consumer.</returns>
+        /// <remarks>https://redis.io/topics/streams-intro</remarks>
+        RedisStreamEntry[] StreamClaimMessages(RedisKey key, RedisValue consumerGroup, RedisValue claimingConsumer, long minIdleTimeInMs, RedisValue[] messageIds, CommandFlags flags = CommandFlags.None);
+
+        /// <summary>
+        /// Change ownership of messages consumed, but not yet acknowledged, by a different consumer. This method returns the IDs for the claimed message(s).
+        /// </summary>
+        /// <param name="key">The key of the stream.</param>
+        /// <param name="consumerGroup">The consumer group.</param>
+        /// <param name="claimingConsumer">The consumer claiming the given message(s).</param>
+        /// <param name="minIdleTimeInMs">The minimum message idle time to allow the reassignment of the message(s).</param>
+        /// <param name="messageIds">The IDs of the messages to claim for the given consumer.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>The message IDs for the messages successfully claimed by the given consumer.</returns>
+        /// <remarks>https://redis.io/topics/streams-intro</remarks>
+        RedisValue[] StreamClaimMessagesReturningIds(RedisKey key, RedisValue consumerGroup, RedisValue claimingConsumer, long minIdleTimeInMs, RedisValue[] messageIds, CommandFlags flags = CommandFlags.None);
+
+        /// <summary>
+        /// Create a consumer group for the given stream.
+        /// </summary>
+        /// <param name="key">The key of the stream.</param>
+        /// <param name="groupName">The name of the group to create.</param>
+        /// <param name="readFrom">The beginning position in the stream from which to read. Pass "$" or <see cref="StreamConstants.NewMessages"/> to read only new messages.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>True if the group was created.</returns>
+        /// <remarks>https://redis.io/topics/streams-intro</remarks>
+        bool StreamCreateConsumerGroup(RedisKey key, RedisValue groupName, RedisValue readFrom, CommandFlags flags = CommandFlags.None);
+
+        /// <summary>
+        /// Retrieve information about the given stream. This is the equivalent of calling "XINFO STREAM key".
+        /// </summary>
+        /// <param name="key">The key of the stream.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>A <see cref="StreamInfo"/> instance with information about the stream.</returns>
+        /// <remarks>https://redis.io/topics/streams-intro</remarks>
+        StreamInfo StreamInfoGet(RedisKey key, CommandFlags flags = CommandFlags.None);
```

Comment:
IMO, this should be just `StreamInfo` (we don't have `Get` as a convention or anything - just in the few places it makes sense).

A few options to consider:
1. `StreamGetInfo`, `StreamGetGroupInfo`, `StreamGetConsumerInfo`
2. `StreamInfo`, `StreamGroupInfo`, `StreamConsumerInfo`
3. `StreamGetInfo`, `StreamGetGroups`, `StreamGetConsumers`

They're all `XINFO` under the covers, and IMO it'd be nice to indicate that in naming if we can - I'm personally a fan of option 2. Thoughts?

Reply:
I like #2 as well.